commands/import_e2e_test.go

```diff
@@ -0,0 +1,309 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestImportUploadCmdF() {
+	s.SetupTestHelper().InitBasic()
+	serverPath := os.Getenv("MM_SERVER_PATH")
+	importName := "import_test.zip"
+	importFilePath := filepath.Join(serverPath, "tests", importName)
+
+	s.Run("no permissions", func() {
+		printer.Clean()
+
+		err := importUploadCmdF(s.th.Client, &cobra.Command{}, []string{importFilePath})
+		s.Require().NotNil(err)
+		s.Require().Equal("failed to create upload: : You do not have the appropriate permissions., ", err.Error())
+		s.Require().Len(printer.GetLines(), 0)
```

Comment:
Would it be safe to use this:
```suggestion
		s.Require().Empty(printer.GetLines())
```

Reply:
Nice catch, we should use Empty in such cases.